core/src/components/input-password-toggle/input-password-toggle.tsx

```diff
@@ -0,0 +1,141 @@
+import type { ComponentInterface } from '@stencil/core';
+import { Component, Element, Host, Prop, h, forceUpdate } from '@stencil/core';
+import { printIonWarning } from '@utils/logging';
+import { createColorClasses } from '@utils/theme';
+import { eyeOff, eye } from 'ionicons/icons';
+
+import { getIonMode } from '../../global/ionic-global';
+import type { Color } from '../../interface';
+
+/**
+ * @virtualProp {"ios" | "md"} mode - The mode determines which platform styles to use.
+ */
+@Component({
+  tag: 'ion-input-password-toggle',
```

Comment:
The component name is quite lengthy. I wanted to use `ion-input` in it to make it clear that it should only be used with `ion-input`. This naming also aligns with the supporting components for `ion-card`. What do we think of that name? I could also be on board with `ion-password-toggle`.

Reply:
Name is fine with me, especially since this _only_ works with `ion-input`. If it worked with any `input`, I'd vote for `ion-password-toggle`.

Reply:
Agree that the name should include `ion-input` since it must be used with `ion-input`.

Reply:
3 in favor of keeping the name as-is, so going to mark as resolved